line_profiler/_line_profiler.pyx

```diff
@@ -339,10 +339,23 @@ cdef class LineProfiler:
                 entries += list(cmap[entry].values())
             key = label(code)
 
-            entries = [(e["lineno"], e["nhits"], e["total_time"]) for e in entries]
+            temp_entries = [(e["lineno"], e["nhits"], e["total_time"]) for e in entries]
             # If there are multiple entries for a line, this will sort them by increasing # hits
-            entries.sort()
-
+            temp_entries.sort()
+            # If a line causes a branch, the stats for the True branch and False branch are separate.
+            # So we merge them here with 2 passes
+            branch_entries = {}
+            # Find the indicies of items with the same line number
```

Comment:
Nit: typo: indices

Reply:
I somehow broke my laptop (it only gets to the GRUB screen, can't even load to the login screen) and am currently on vacation, so I'll try and fix it when I get home within a few days. Do you think this is good to go otherwise, and this typo can be fixed in a later PR?

Reply:
This typo is actually obsoleted by my suggestion below.